pyproject.toml

```diff
@@ -42,4 +47,10 @@ build-backend = "poetry.core.masonry.api"
 [tool.poetry.scripts]
 mint = "cashu.mint.main:main"
 cashu = "cashu.wallet.cli:cli"
-wallet-test = "tests.test_wallet:test"
\ No newline at end of file
+wallet-test = "tests.test_wallet:test"
+
+[tool.coverage.run]
+source = ["cashu"]
+
+[tool.black]
+line-length = 120
```

Comment:
Isn't this the default setting?

Reply:
I think the default is `88` https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length
I personnally find 88 too short. Another advantage of specifying here, is to let other contributors know what the projects expects. Again, can change if you prefer another length